src/@chakra-ui/gatsby-plugin/foundations/colors.ts

```diff
@@ -22,10 +22,15 @@ const colors = {
     800: "#451900",
   },
   red: {
+    100: "#F7C8C8",
     500: "#b80000",
+    900: "#1B0C0C",
   },
   green: {
+    100: "#C8F7D8",
+    400: "#48BB78",
```

Comment:
I know this is used for success in dark mode. but this breaks the design system currently. @nloureiro @konopkja just calling out that this should get added in design system colors as it seems like the direction we are going with designs.

Reply:
https://github.com/ethereum/ethereum-org-website/pull/8094#discussion_r1005036237

Duplicate comment, sorry. What @wackerow said.

Reply:
I saw your comment on Figma. We need to do a revision on the quizzes colours after the launch.
For now it's good on my end